### PR TITLE
Website update/maintenance - Dev branch pull request

### PR DESCRIPTION
**Pull request by ivanzinho09**
Performing a pull request to try and get this website maintenance and upgrade move faster!
Please provide feedback on the work done so far, and what still needs to be added.

### CODE REVIEW

**Comment by ivanzinho09** [comment]
**Note:** There seems to be a conflict with the read me file, and it might be the fact that it is 1 AM, but I tried to fix it in my git terminal, but haven't been able to figure it out.

**Comment by JJ11teen** [comment]
I can look into the conflict tomorrow, don't worry. I'll spend some time reviewing too.

**Comment by JJ11teen** [comment]
Linking #2

**Comment by ivanzinho09** [comment]
@JJ11teen , I ended up solving it, I think :) 
Excited for the feedback tomorrow!

File: About.html [hunk 1]
```diff
@@ -68,20 +68,93 @@
...
-    <p><span class="w3-large">engages communities and provides education</span></p>
-    <p><span class="w3-large">about Blockchain technology.</span></p>
-    <p><a href="#about" class="w3-button w3-white w3-padding-large w3-large w3-margin-top w3-opacity w3-hover-opacity-off">Learn more and start today</a></p>
-  </div> 
+   <div class="bgimg-1 ui vertical segment" style="padding: 8em 0em;">
+                    <div class="ui container stackable middle aligned grid">
+                        <div class="row">
+                            <div class="column">
+                                <h3 class="ui header" style="font-size: 2em;">What is this all about?</h3>
+                                <p style="font-size: 1.2em;"><strong>Blockchain</strong> is a technology that provides true peer-to-peer transfer of value. It removes the need to trust intermediaries and third parties to verify validity of transactions.</p>
```
Comment (JJ11teen) [inline About.html]: These inline font sizes should be moved to stylesheet

Comment (JJ11teen) [reply About.html]: In general all of the inline styles should be moved to stylesheet

**Comment by JJ11teen** [comment]
I started adding some code linting issues but I think in general there are more important dev chore things:

* folder structure should follow a standard (usually /js /styles and /assets directories, with html in root
* files need to be meaningfully named (eg background png), and follow consistent (usually camel) casing
* add a readme file documenting external libraries being used and why/what for

From a usability perspective:

* The 'sponsors' tab on the navbar has a larger highlight area than the other tabs
* mobile support needs work, padding missing around text and buttons not aligned
* white text is difficult to read over background with certain alignments
* social media buttons overlap sponsors on small screens

**Review by JJ11teen — CHANGES_REQUESTED** [review]
Small nits along with larger issues commented earlier. Seems some file directory stuff has been fixed by now though which is good. A readme explaining the libraries is a must however.

File: About.html (hunk 1 above)
Comment (ooyendyk) [reply About.html]: Good idea, I'll move 'em across.

**Comment by ooyendyk** [comment]
On the subject of the read me: There's a few libraries that are not currently in use and will probably never be used (i.e. wow.js, simplyCountdown ect.). Should I just say that they are unused or would It be worth deleting them altogether?

**Comment by ivanzinho09** [comment]
Awesome work on the changes to the file and folder structure and the updated about section, @nerd-1 !

The social media icons you fixed in the sponsor's page though, whilst looking better for tablet and desktop users, unfortunately do not function properly for the mobile version of the website (from testing done this afternoon). 

> On the subject of the read me: There's a few libraries that are not currently in use and will probably never be used (i.e. wow.js, simplyCountdown ect.). Should I just say that they are unused or would It be worth deleting them altogether?

I think that for now, mainly focusing on the layout and adding in content would be probably the better. In the future though and just in general, if we are not going to use them, might as well delete them. Don't stress too much over it for now though :)

**Comment by paulpounendis** [comment]
Good work guys, I will plan to do some graphic design changes today. I pesonally hate the font haha ... If we can get the resources page up I can start going along and chucking some content in.

**Comment by JJ11teen** [comment]
@nerd-1 deleting if they are unused would be great. 
Sorry @ivanzinho09 but I've got to disagree with your comment on current focus. A readme doc is invaluable in increasing development productivity and should be prioritised appropriately. Content and style are important, but a readme should document global styles and patterns used and result and faster turn around on a more functional, more consistent product here.

**Comment by paulpounendis** [comment]
As a fyi, I am looking to eventually get the UI looking super clean. I can make it alot nicer, and will plan to make it have the same impact as this site for example. 

https://tenzorum.org/

**Comment by ivanzinho09** [comment]
> Sorry @ivanzinho09 but I've got to disagree with your comment ....

@JJ11teen, fair enough, actually. That does seem like a better and more consistent method of working on things. Let's get do it that way instead then :)

**Comment by ivanzinho09** [comment]
@nerd-1, the page keeps getting better and better! Sorry I haven't been able to actively develop a lot more for it, but I will likely change up the contact section (whenever I can!!) to the one we already have set up in our dreamhost-surveymonkey account ;) It just looks a bit cleaner aha.

Reference for layout and theme scheme: 
https://puu.sh/BFDf2/27686c4434.png

**Pull request merged**